pkgs/development/libraries/gtk-layer-shell/default.nix

```diff
@@ -35,11 +52,12 @@ stdenv.mkDerivation rec {
     docbook-xsl-nons
     docbook_xml_dtd_43
     vala
+    wayland-scanner
   ];
 
   buildInputs = [
-    wayland
     gtk3
+    gtk-doc
```

Comment:
https://github.com/wmww/gtk-layer-shell/blob/3073ff944ad19e8c3db5243bb9079f3fc54a23cb/doc/meson.build#L7 should be removed. `gnome.gtkdoc` already does its own checking